lib/active_record/connection_adapters/postgis_adapter.rb

```diff
@@ -90,3 +90,21 @@ def quote(value)
     end
   end
 end
+
+# if using JRUBY, create ArJdbc::PostGIS module
+# and prepend it to the PostgreSQL adapter since
+# it is the default adapter_spec.
+# see: https://github.com/jruby/activerecord-jdbc-adapter/blob/master/lib/arjdbc/postgresql/adapter.rb#27
+if RUBY_ENGINE == "jruby"
+  module ArJdbc
+    module PostGIS
+      ADAPTER_NAME = 'PostGIS'.freeze
```

Comment:
you should use the top comment `# frozen_string_literal: true` rather than having to freeze every string :)